template_feed/Microsoft.DotNet.Test.ProjectTemplates.2.1/content/NUnit-VisualBasic/Company.TestProject1.vbproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
     <PackageReference Include="NUnit" Version="3.13.3" />
```

Comment:
No updates are out for NUnit?

Reply:
Nope, checked on nuget.org, latest version for `NUnit` is 3.13.3. There's some alpha versions for `NUnit3TestAdapter` but I preferred to stick to the stable 4.2.1 version.